src/components/savedJobs/SavedJobs.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import Tabs from 'react-bootstrap/Tabs';
+import Tab from 'react-bootstrap/Tab';
+import TopNav from '../topNav/TopNav.jsx';
+
+const SavedJobs = props => {
+  const [key, setKey] = useState('saved');
+  const [navItems, setNavItems] = useState(['Home', 'Calendar', 'Notes', 'Login/Logout']);
+
+  return (
+    <div>
+      <TopNav navItems={navItems} />
+      <div id="tabs">
+        <Tabs
+          activeKey={key}
+          onSelect={(k) => setKey(k)}
+        >
+          <Tab eventKey="saved" title="Saved">
```

Comment:
Maybe condense this with a for loop:

Something like declare a variable:

var tabs = ['Saved', 'Applied', 'Interview']

Then down in your react code:

{tabs.map((tab) => {
return (
<Tab eventkey={tab} title={tab}>
<div className="tabs>
<h2>Job Title</h2>
<section>Description</section>
<section>Status</section>
</div>
</Tab>
)})}

Note sure if that's totally syntactically correct, but you get the idea